### DISCUSSION TITLE
ようこそ 63rd Antarctic Sprint (63ants) へ！

### DISCUSSION THREAD

**Discussion by hfu** [optgeo/63ants#1] [Announcements]
## 👋 ようこそ！
コミュニティのメンバーとつながるために Discussions を使っています。こんなことをしていきましょう。
  * 気になることを質問する。
  * アイディアを共有する。
  * コミュニティのメンバーと触れ合う。
  * みんなを歓迎してよく話を聞きましょう。ともに作るコミュニティであることを忘れずに💪.

手始めに、ここのコメント欄に自己紹介をして、このコミュニティで何をするか教えてください。

## 🐧その前に、ここでは何をするの？
[第６３次南極地域観測隊](https://www.mext.go.jp/content/20210630-mxt_kaiyou-000015939_2.pdf)越冬隊をご支援して、Raspberry Pi 4B を使って南極でベクトルタイルのウェブ地図を使えるようにします！

**Comment by hfu** [comment]
こんにちは。藤村(@hfu)です。こんなことをしていきたいと思っています。

1. [国土地理院の「南極の地理空間情報」にある Shapefile 形式のファイルからベクトルタイルデータを作成する](https://github.com/optgeo/63ants/discussions/2)。
2. [Raspberry Pi (4B) に USB 接続した簡易な GPS アンテナから得られる概略の位置情報を利用する方法を整理する](https://github.com/optgeo/63ants/discussions/3)。

@SawagakiT のご指導を頂いています！

**Reply by SawagakiT** [reply]
63次南極観測隊越冬隊長の澤柿（@SawagakiT）です．どうぞよろしくお願いします．
